sycl/include/CL/sycl/item.hpp

```diff
@@ -26,24 +29,39 @@ template <int dimensions = 1, bool with_offset = true> class item {
 public:
   item() = delete;
 
-  id<dimensions> get_id() const { return MImpl.MIndex; }
+  id<dimensions> ALWAYS_INLINE get_id() const { return MImpl.MIndex; }
 
-  size_t get_id(int dimension) const { return MImpl.MIndex[dimension]; }
+  size_t ALWAYS_INLINE get_id(int dimension) const {
+    size_t id = MImpl.MIndex[dimension];
```

Comment:
One more minor comment. I don't remember if we follow LLVM conding standard. If we do, then variables should start with a capital letter

CodingStandards.rst :
> * **Variable names** should be nouns (as they represent state).  The name should
  be camel case, and start with an upper case letter (e.g. ``Leader`` or
  ``Boats``).

Reply:
Fixed.